electron/index.js

```diff
@@ -45,18 +49,10 @@ function createWindow () {
 
   mainWindow = new BrowserWindow({
     height: 800,
-    width: 1200
+    width: 1200,
+    webPreferences: { nodeIntegrationInWorker: true }
```

Comment:
Nodeintegration is enabled for the BrowserWindow but disabled by default for its WebWorkers; I had to add this option to remove the following error (wasn't blocking though):

![image](https://user-images.githubusercontent.com/5739676/41736623-5e9faf60-758d-11e8-8272-88cd70c769a0.png)

Reply:
Should be fine, we are not requiring any external files as web workers.

Is the error stack longer? Who is requiring `@parity/shared/lib/webWorker.js`? I'd be more enthusiast to remove web workers altoghether.

Reply:
that's the whole error stack

we import some files of `@parity/shared` (for example `import { initStore } from '@parity/shared/lib/redux'` in `index.parity.js`) that require the `worker` of `@parity/shared` (in that case, `initStore` calls `setupWorker`). I imagine that's where it's coming from